src/v2/providers/alerts/appDistribution.ts

```diff
@@ -0,0 +1,104 @@
+import { defineEndpoint, FirebaseAlertData } from '.';
```

Comment:
nit* I would've prefer having this point to a file instead, e.g. './alerts`. Saves me one jump to get to the actual source.

Reply:
fixed